RELEASE.md

```diff
@@ -30,6 +31,7 @@
 ## Thanks for supporting contributions
 [Moussa Taifi](https://github.com/moutai),
 [Deepyaman Datta](https://github.com/deepyaman)
+[Benjamin Levy](https://github.com/BenjaminLevyQB)
```

Comment:
```suggestion
[Deepyaman Datta](https://github.com/deepyaman),
[Benjamin Levy](https://github.com/BenjaminLevyQB)
```

Reply:
(these linebreaks get squashed out when the markdown is rendered so we need the comma too)